.github/workflows/performance-tests.yml

```diff
@@ -36,12 +36,12 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # https://github.com/microsoft/vscode-ripgrep/issues/9
 
       - name: Performance (browser)
-        uses: GabrielBB/xvfb-action@v1
+        uses: coactions/setup-xvfb@v1
```

Comment:
for the browser, `xvfb` should not be needed? At least when I run this test suite locally, I see no window being created.

Reply:
I agree, for browser we do not need xvfb.

Reply:
Right, I think so too. I just kept it in as it was there before. But you're right, let's remove it.